engines/cine/main_loop.cpp

```diff
@@ -166,24 +175,39 @@ static void processEvent(Common::Event &event) {
 		case Common::KEYCODE_KP3:
 			moveUsingKeyboard(+1, -1); // Down & Right
 			break;
+		case Common::KEYCODE_LEFT:
```

Comment:
Maybe it is too much to ask, but if you could take a look into the new keymapper code (see Wintermute engine), that would be awesome.

Reply:
Ah, here is how to do it: https://wiki.scummvm.org/index.php?title=Keymapper

Reply:
Yes, this is too much to ask at this point. So, not taken into account.